recipes/glib/all/conanfile.py

```diff
@@ -1,14 +1,17 @@
-import functools
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.scm import Version
+from conan.tools.microsoft import is_msvc
+from conan.tools.meson import Meson, MesonToolchain
+from conan.tools.gnu import PkgConfigDeps, AutotoolsDeps
+from conan.tools.files import apply_conandata_patches, get, replace_in_file, rmdir, chdir, rm, load, save, copy
+# TODO: Update to conan.tools.apple after Conan 1.51.3
+from conans.tools import is_apple_os
+import re
 import os
 import glob
 import shutil
 
-from conan import ConanFile
-from conan.errors import ConanInvalidConfiguration
-from conan.tools.build import cross_building
-from conan.tools import files, scm
-from conan.tools.microsoft import is_msvc
-from conans import tools, Meson, VisualStudioBuildEnvironment
 
 required_conan_version = ">=1.50.0"
```

Comment:
```suggestion
required_conan_version = ">=1.51.0"
```

Does it require any specific patch version?